java/core/src/java/org/apache/orc/impl/TreeReaderFactory.java

```diff
@@ -316,12 +335,13 @@ protected long countNonNulls(long rows) throws IOException {
      * @param batchSize      Size of the column vector
      * @param filterContext the information about the rows that were selected
      *                      by the filter.
+     * @param rLevel The read level
      * @throws IOException
      */
     public void nextVector(ColumnVector previous,
                            boolean[] isNull,
                            final int batchSize,
-                           FilterContext filterContext) throws IOException {
+                           FilterContext filterContext, ReadLevel rLevel) throws IOException {
```

Comment:
In this case, shall we add this to the next line?

Reply:
sure, changed